multi threading/performance_comparison.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
```

Comment:
can you move the code inside multi_threading directory?

Reply:
@t2013anurag I fixed it in my fork, but it looks like maybe the change hasn't propagated to the PR yet...

https://github.com/Jdsleppy/programming/tree/jdsleppy-multithreading-in-python/multi_threading